TramsDataApi/DatabaseModels/TramsDbContext.cs

```diff
@@ -752,6 +756,31 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                 .WithOne()
                 .OnDelete(DeleteBehavior.Cascade);
             });
+
+
+            modelBuilder.Entity<Decision>(e =>
+            {
+                e.ToTable("ConcernsDecision", "sdd");
+                e.HasKey(x => x.DecisionId);
+                e.Property(x => x.TotalAmountRequested).HasColumnType("money");
+                e.HasMany(x => x.DecisionTypes).WithOne();
+            });
+
+            modelBuilder.Entity<DecisionType>(e =>
+            {
+                e.ToTable("ConcernsDecisionType", "sdd");
+                e.HasKey(x => new { x.DecisionId, x.DecisionTypeId });
+            });
+
+            modelBuilder.Entity<DecisionTypeId>(e =>
+            {
+                e.ToTable("ConcernsDecisionTypeId", "sdd");
+                e.HasKey(x => x.Id);
+                e.HasData(
+                    Enum.GetValues(typeof(Enums.Concerns.DecisionType)).Cast<Enums.Concerns.DecisionType>()
```

Comment:
If we add or change a decisiontype enum, does this code mean it automatically knows to include that update when the migration is generated? (If so, very cool!)

Reply:
I believe so